.github/ISSUE_TEMPLATE/1-bug-report.md

```diff
@@ -0,0 +1,56 @@
+---
+name: "\U0001F41EBug report"
+about: Report a bug in webhint
+---
+<!--
+
+Hi there 👋
+
+We are sorry you've found an issue in our project. Before
```

Comment:
Would be fun to frame this more positively: "Thanks for taking the time to make webhint better!"